diff --git a/node_modules/@react-native-voice/voice/android/build.gradle b/node_modules/@react-native-voice/voice/android/build.gradle
index 8fce879..243c620 100644
--- a/node_modules/@react-native-voice/voice/android/build.gradle
+++ b/node_modules/@react-native-voice/voice/android/build.gradle
@@ -1,17 +1,26 @@
 apply plugin: 'com.android.library'
 
+
 repositories {
-    mavenLocal()
-    jcenter()
+//     mavenLocal()
+//    mavenCentral()
+//     maven {
+//         // For developing the library outside the context of the example app, expect `react-native`
+//         // to be installed at `./node_modules`.
+//         url "$projectDir/../node_modules/react-native/android"
+//     }
+//     maven {
+//         // For developing the example app.
+//         url "$projectDir/../../react-native/android"
+//     }
+mavenLocal()
+    mavenCentral()
+    google()
     maven {
         // For developing the library outside the context of the example app, expect `react-native`
         // to be installed at `./node_modules`.
         url "$projectDir/../node_modules/react-native/android"
     }
-    maven {
-        // For developing the example app.
-        url "$projectDir/../../react-native/android"
-    }
 }
 
 def DEFAULT_COMPILE_SDK_VERSION = 28
@@ -20,15 +29,22 @@ def DEFAULT_TARGET_SDK_VERSION = 28
 def DEFAULT_SUPPORT_LIB_VERSION = "28.0.0"
 
 android {
+    compileSdkVersion safeExtGet('compileSdkVersion', 34)
+    buildToolsVersion safeExtGet('buildToolsVersion', "34.0.0")
     compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : DEFAULT_COMPILE_SDK_VERSION
     buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
 
     defaultConfig {
-        minSdkVersion 15
-        targetSdkVersion rootProject.hasProperty('targetSdkVersion') ? rootProject.targetSdkVersion : DEFAULT_TARGET_SDK_VERSION
+       minSdkVersion safeExtGet('minSdkVersion', 21)
+        targetSdkVersion safeExtGet('targetSdkVersion', 34)
         versionCode 1
         versionName "1.0"
     }
+
+    lintOptions {
+        abortOnError false
+    }
+
     buildTypes {
         release {
             minifyEnabled false
@@ -57,11 +73,20 @@ allprojects {
     }
 }
 
-def supportVersion = rootProject.hasProperty('supportLibVersion') ? rootProject.supportLibVersion : DEFAULT_SUPPORT_LIB_VERSION
-
+// def supportVersion = rootProject.hasProperty('supportLibVersion') ? rootProject.supportLibVersion : DEFAULT_SUPPORT_LIB_VERSION
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
 dependencies {
+    // implementation fileTree(dir: 'libs', include: ['*.jar'])
+    // testImplementation 'junit:junit:4.12'
+    // implementation "com.android.support:appcompat-v7:${safeExtGet('supportLibVersion', '28.0.0')}"
+    // implementation 'com.facebook.react:react-native:+'
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "com.facebook.react:react-native:+"
+    
+    // Updated to AndroidX (Support libraries are deprecated)
+    implementation "androidx.appcompat:appcompat:1.1.0"
+    
     testImplementation 'junit:junit:4.12'
-    implementation "com.android.support:appcompat-v7:${supportVersion}"
-    implementation 'com.facebook.react:react-native:+'
 }
